src/Microsoft.Azure.WebJobs.Host/Indexers/FunctionIndexer.cs

```diff
@@ -372,9 +363,6 @@ private bool HasJobAttribute(CustomAttributeData attributeData)
         {
             var disabled = HostListenerFactory.IsDisabled(method, nameResolver, jobActivator);
 
-            // Determine the TraceLevel for this function (affecting both Console as well as Dashboard logging)
-            TraceLevelAttribute traceAttribute = TypeUtility.GetHierarchicalAttributeOrNull<TraceLevelAttribute>(method);
```

Comment:
Why are we not supporting this anymore?

Reply:
I mentioned it above -- it's handled via the Filters you can build. You can control each category -- so you can tweak individual function levels.